assets/js/index.js

```diff
@@ -0,0 +1,76 @@
+function WeatherService(API_ID) {
+	this.API = API_ID;
+	this.getForecast = this.getForecast.bind(this);
+	this.getWeather = this.getWeather.bind(this);
+}
+
+
+WeatherService.prototype.getWeather = function (city, callback) {
+	var request = new XMLHttpRequest();
+	request.open("GET", `https://api.openweathermap.org/data/2.5/weather?q=${city}&APPID=${this.API}`, true);  // `false` makes the request synchronous
+	request.onload = function () {
+		callback(JSON.parse(this.response))
+	};
+	request.send(null);
+}
+
+WeatherService.prototype.getForecast = function (city_ID, numDays, callback) {
+	var request = new XMLHttpRequest();
+	request.open("GET", `https://api.openweathermap.org/data/2.5/forecast?id=${city_ID}&cnt=${numDays}&APPID=${this.API}`, true);  // `false` makes the request synchronous
+	request.onload = function () {
+		callback(JSON.parse(this.response))
+	};
+	request.send(null);
+}
+
```

Comment:
Let's change prototype declaration to object syntaxis like: 
```
WeatherService.prototype = {
    getWeather: function(...) {},
    getForecast: function(...) {}
}
```
Above is closer to what we are gonna see next.